### PR TITLE
add contexts image other updates

### PR DESCRIPTION
**Pull request by rosieyohannan**
Updates to Contexts guide inc:

- [x] overview image
- [x] fix up approval job section and add a code example

### CODE REVIEW

File: jekyll/_cci2/contexts.md [hunk 1]
```diff
@@ -7,104 +7,124 @@ categories: [configuring-jobs]
...
 ### Restrict a Context to a Security Group or Groups
+{:.no_toc}
 
 You must be an organization administrator to complete the following task.
 
 1. Navigate to Organization Settings > Contexts page in the CircleCI app. The list of contexts appears. The default security group is `All members` and allows all users in the org to invoke jobs with that context.
-2. Click the Create Context button if you will use a new context or click the name of an existing context.
-3. Click the Add Group link. The Add Groups dialog box appears.
+2. Click the Create Context button if you wish to use a new context, or click the name of an existing context.
+3. Click the Add Secutrity Group button to view the dialog bo.
```
Comment (thomasr8) [inline jekyll/_cci2/contexts.md:78]:
```suggestion
3. Click the Add Security Group button to view the dialog box.
```

Comment (rosieyohannan) [reply jekyll/_cci2/contexts.md:78]: wowzers you can tell my head is in two places at once, thanks Thomas!

File: jekyll/_cci2/contexts.md [hunk 2]
```diff
@@ -7,104 +7,124 @@ categories: [configuring-jobs]
...
 * TOC
 {:toc}
 
-Contexts provide a mechanism for securing and sharing environment variables across projects. The environment variables are defined as name/value pairs and are injected at runtime.
-
 ## Overview
 {:.no_toc}
 
-Contexts are created on the Settings page of the CircleCI application, in the Organization section. You must be an organization member to view, create, or edit contexts. After a context is set in the application it may be configured in the workflows section of the [`config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file for a project.
+Create and manage contexts on the Organization Settings page of the CircleCI application. You must be an organization member to view, create, or edit contexts. After a context has been created, you can use the `context` key in the workflows section of a project [`config.yml`]({{ site.baseurl }}/2.0/configuration-reference/#context) file, to give any job(s) access to the environment variables associated with the context, as shown in the image below.
```
Comment (rosieyohannan) [inline jekyll/_cci2/contexts.md:18]:
```suggestion
Create and manage contexts on the Organization Settings page of the CircleCI application. You must be an organization member to view, create, or edit contexts. After a context has been created, you can use the `context` key in the workflows section of a project [`config.yml`]({{ site.baseurl }}/2.0/configuration-reference/#context) file to give any job(s) access to the environment variables associated with the context, as shown in the image below.
```

File: jekyll/_cci2/contexts.md [hunk 3]
```diff
@@ -7,104 +7,136 @@ categories: [configuring-jobs]
...
+{:.no_toc}
 
-Adding an approval job to a workflow allows a user to manually approve the use of a restricted context. 
-
-To restrict running of jobs that are downstream from an approval job, add a restricted context to those downstream jobs. 
+Adding an [approval job]({{ site.baseurl }}/2.0/configuration-reference/#type) to a workflow gives the option to require manual approval of the use of a restricted context. To restrict running of jobs that are downstream from an approval job, add a restricted context to those downstream jobs, as shown in the example below:
 
-For example, if you want the execution of job C and job D restricted to a security group, you need to add an approval job B before the jobs C and D to that use a context with a security group. That is, you can have four jobs in a workflow, job A can run unrestricted, the approval job B may be approved by any member, but the jobs C and D after the approval may only be executed by someone in the security group for the context used on jobs C and D. 
+```yaml
+workflows:
```
Comment (teesloane) [inline jekyll/_cci2/contexts.md:101]: As this is a partial example (not displaying the jobs that are referenced) I would maybe add a commend above the workflows that indicates other text is expected to exist (`# ...`)

Comment (rosieyohannan) [reply jekyll/_cci2/contexts.md:101]: Oh yes good idea 👍

File: jekyll/_cci2/contexts.md [hunk 4]
```diff
@@ -7,104 +7,140 @@ categories: [configuring-jobs]
...
 
-Adding an approval job to a workflow allows a user to manually approve the use of a restricted context. 
+Adding an [approval job]({{ site.baseurl }}/2.0/configuration-reference/#type) to a workflow gives the option to require manual approval of the use of a restricted context. To restrict running of jobs that are downstream from an approval job, add a restricted context to those downstream jobs, as shown in the example below:
 
-To restrict running of jobs that are downstream from an approval job, add a restricted context to those downstream jobs. 
+```yaml
+version: 2.1
 
-For example, if you want the execution of job C and job D restricted to a security group, you need to add an approval job B before the jobs C and D to that use a context with a security group. That is, you can have four jobs in a workflow, job A can run unrestricted, the approval job B may be approved by any member, but the jobs C and D after the approval may only be executed by someone in the security group for the context used on jobs C and D. 
+# jobs declaration from build, test and deploy not displayed for brevity
```
Comment (rosieyohannan) [inline jekyll/_cci2/contexts.md:103]:
```suggestion
# jobs declaration for build, test and deploy not displayed
```

**Pull request merged**